src/storage/index.js

```diff
@@ -452,4 +452,82 @@ export function deleteFile(path: string) {
   Promise.reject(new Error(`Delete of ${path} not supported by gaia hubs`))
 }
 
+/**
+ * Loop over the list of files in a Gaia hub, and run a callback on each entry.
+ * Not meant to be called by external clients.
```

Comment:
We should mark this as `@private` so that it doesn't show up in the public documentation.

Reply:
Done